CONTRIBUTING.md

```diff
@@ -67,8 +67,12 @@ public extension SomeType {
 	- Properties & static properties
 	- Methods & static methods
 	- Initializers
-- Files are named based on the type that the contained extensions extend (example: all String extensions are found in "**StringExtensions.swift**" file)
-- extensions are ordered inside files in the following order:
+- Files are named based on the type that the contained extensions extend. (example: all String extensions are found in "**StringExtensions.swift**" file)
+- All extensions should be tested.
+- All extensions files and test file have a one to one relation.
```

Comment:
Looks like you might have a typo here. Should it be "test files"? You might also want to explain what it means for the files to have a one to one relation, but it could go either way ¯\_(ツ)_/¯.

Reply:
Agree on the type and explanation. I wasn't sure since it's more or less explained right in the next line but ¯\_(ツ)_/¯